packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/ClientProvider.cs

```diff
@@ -192,7 +192,7 @@ private IReadOnlyList<ParameterProvider> GetClientParameters()
             return null;
         }
 
-        internal RestClientProvider RestClient => _restClient ??= new RestClientProvider(_inputClient, this);
+        public RestClientProvider RestClient => _restClient ??= new RestClientProvider(_inputClient, this);
```

Comment:
Please add ref docs now that this is public.

Reply:
added